etcdmain/help.go

```diff
@@ -161,7 +161,18 @@ given by the consensus protocol.
 profiling flags:
 	--enable-pprof 'false'
 		Enable runtime profiling data via HTTP server. Address is at client URL + "/debug/pprof/"
+
 	--metrics 'basic'
 	  Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
+
+auth flags:
+	--auth-token-type 'simple'
```

Comment:
Maybe this should be one big flag `--auth-token jwt,sign=...,pubkey=...,privkey=...` / `--auth-token simple` to avoid having no-op flags is `simple` is enabled? There could be an auth package function that converts the config string to the token provider.